src/android/index.jsx

```diff
@@ -260,7 +260,7 @@ class Android extends Component {
                         {
                           eq: {
                             framework: 10,
-                            repo: 'mozilla-central',
+                            repo: MOZILLA_CENTRAL,
```

Comment:
@Je-ni  let's consider this specific series:

```
                series={[
                  {
                    label: 'Moto G5 (arm7)',
                    filter: {
                      and: [
                        { missing: 'test' },
                        {
                          prefix: { platform: 'android-hw-g5-7-0-arm7-api-16' },
                        },
                        {
                          eq: {
                            framework: 10,
                            repo: 'mozilla-central',
   ```

Add the `repo` property to the  series, and leave the filter alone

```
                series={[
                  {
                    label: 'Moto G5 (arm7)',
                    repo: MOZILLA_CENTRAL,
                    filter: {
                      and: [
                        { missing: 'test' },
                        {
                          prefix: { platform: 'android-hw-g5-7-0-arm7-api-16' },
                        },
                        {
                          eq: {
                            framework: 10,
                            repo: 'mozilla-central',
```

> In a situation where the dataset is an array instead of an object as in

In the array situation,

```
const COMBO_TABLE = {
  header: ['browser', 'browserLabel', 'suite', 'suiteLabel', 'filter'],
  data: [
    ['geckoview', 'Geckoview', 'scn-power-idle', 'about:blank page', {eq: {framework: 10, repo: 'mozilla-central', suite: ['raptor-scn-power-idle-geckoview-power']}}],
```

you must also add the header, while leaving the `filter` unchanged

```
const COMBO_TABLE = {
  header: [`repo`,  'browser', 'browserLabel', 'suite', 'suiteLabel', 'filter'],
  data: [
    [MOZILLA_CENTRAL, 'geckoview', 'Geckoview', 'scn-power-idle', 'about:blank page', {eq: {framework: 10, repo: 'mozilla-central', suite: ['raptor-scn-power-idle-geckoview-power']}}],
```

in both cases we have added a property, but we are still not using it.  There is still some we must add to use this new property; at least all the existing code will continue to running.